packages/components/src/Actions/ActionDropdown/ActionDropdown.snapshot.test.js

```diff
@@ -179,4 +179,20 @@ describe('ActionDropdown', () => {
 		// then
 		expect(wrapper.html()).toMatchSnapshot();
 	});
+
+	it('should apply transformation on icon', () => {
+		// given
+		const props = {
+			id: 'dropdown-id',
+			label: 'Button label',
+			icon: 'talend-ellipsis',
+			iconTransform: 'rotate-90',
+		};
+
+		// when
+		const wrapper = mount(<ActionDropdown {...props} />).find('DropdownButton');
+
+		// then
+		expect(wrapper.html()).toMatchSnapshot();
```

Comment:
PLease look for the svg.theme-rotate-90 element to validate the behavior. Not a complete snapshots for that